### PR TITLE
Classes & Props: Container/Grid/Row/Column

### PR DESCRIPTION
**Pull request by levithomason**
Only annotated the first two files, the others have the same changes.

### CODE REVIEW

File: src/components/Container/Container.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import classNames from 'classnames';
  * </Container>
  */
 
-class Container extends Component {
+export default class Container extends Component {
```
Comment (levithomason) [inline src/components/Container/Container.js:15]: inline export

File: src/components/Container/Container.js [hunk 2]
```diff
@@ -21,16 +21,14 @@ class Container extends Component {
   render() {
     let classes = classNames(
       'sd-container',
-      this.props.className,
       'ui',
-      'container'
+      this.props.className,
+      'container',
```
Comment (levithomason) [inline src/components/Container/Container.js:26]: insert className between `ui` and the component name class.

File: src/components/Container/Container.js [hunk 3]
```diff
@@ -21,16 +21,14 @@ class Container extends Component {
...
     );
     return (
-      <div className={classes}>
+      <div {...this.props} className={classes}>
```
Comment (levithomason) [inline src/components/Container/Container.js:29]: spread props

File: src/components/Grid/Column.js [hunk 4]
```diff
@@ -2,7 +2,7 @@ import classNames from 'classnames';
 import React, {Component, PropTypes} from 'react';
 import numberToWord from 'src/utils/numberToWord';
 
-class Column extends Component {
+export default class Column extends Component {
```
Comment (levithomason) [inline src/components/Grid/Column.js:5]: inline export

File: src/components/Grid/Column.js [hunk 5]
```diff
@@ -13,15 +13,14 @@ class Column extends Component {
   render() {
     let classes = classNames(
       'sd-column',
+      this.props.className,
```
Comment (levithomason) [inline src/components/Grid/Column.js:16]: inherit classes

File: src/components/Grid/Column.js [hunk 6]
```diff
@@ -13,15 +13,14 @@ class Column extends Component {
...
       this.props.width && numberToWord(this.props.width) + ' wide',
       'column'
     );
     return (
-      <div className={classes} style={this.props.style}>
+      <div {...this.props} className={classes}>
```
Comment (levithomason) [inline src/components/Grid/Column.js:21]: spread props

**Comment by athurman** [comment]
:ocean: :octopus:

**Pull request merged**